hashicorp/web-unified-docs
# Chore: remove HCP docs content

# Changes

- Remove hcp-docs content from code base to prep for education engineer test. This includes:
  - Removing hcp-docs from the `content` dir
  - Removing hcp-docs from the `public/assets` dir
- Add `boundary`, and `consul` back to `scripts/migrate-content/repo-config.mjs` per Ruben's request [here](https://hashicorp.slack.com/archives/C07ERKA7MTJ/p1729181416916279)
- Add `boundary`, and `consul` back to `app/api/content/[productSlug]/doc/[version]/[...docsPath]/route.ts` per Ruben's request [here](https://hashicorp.slack.com/archives/C07ERKA7MTJ/p1729181416916279)
  
- [Asana task](https://app.asana.com/0/1207899860738460/1208561647881350/f)

# Testing

1. Make a curl request or use Postman to send a `GET` request 
  ```
  curl https://web-presence-experimental-docs.vercel.app/api/content/hcp-docs/nav-data/latest/docs
  ```
2. You should receive a valid response, such as this:
<img width="1137" alt="Screenshot 2024-10-16 at 15 27 18" src="https://github.com/user-attachments/assets/7b04a4e8-916b-4a27-849b-98421783d340">


3. Make a curl request or use Postman to send a `GET` request
```
curl https://web-presence-experimental-docs-git-heat-chorer-9d2822-hashicorp.vercel.app/api/content/hcp-docs/nav-data/latest/docs
```
4. You should receive a response of `Not Found`
